tensorboard/default.py

```diff
@@ -86,4 +87,30 @@ def get_plugins():
 
   :rtype: list[Union[base_plugin.TBLoader, Type[base_plugin.TBPlugin]]]
   """
+
   return _PLUGINS[:]
+
+# TODO(stephanwlee): Combine with get_plugins when concept of first-party
+# plugins is undone.
+def get_dynamic_plugins():
+  """Returns a list specifying TensorBoard's dynamically loaded plugins.
+
+	A dynamic TensorBoard plugin is specified using entry_points [1]. This will be
+  the de facto mechanism to load plugins and there no longer will be a concept
+  of the first-party plugins as depicted above in `get_plugins`.
+
+  This list can be passed to the `tensorboard.program.TensorBoard` API.
+
+  Returns:
+		a list of base_plugin.TBPlugin
```

Comment:
They’ll actually be either `TBPlugin`s or `TBLoader`s, right? I.e., the
return type is the same as that of `get_plugins`?